src/Umbraco.Core/Configuration/Models/SecuritySettings.cs

```diff
@@ -59,6 +59,11 @@ public class SecuritySettings
     /// </summary>
     public bool UsernameIsEmail { get; set; } = true;
 
+    /// <summary>
+    ///     Gets or sets a value indicating whether the member's email address must be unique.
+    /// </summary>
+    public bool RequireUniqueEmailForMembers { get; set; } = true;
```

Comment:
The other member-specific settings all use a prefix, to this should probably be renamed to `MemberRequireUniqueEmail` if we want to align with those...